src/app/pages/not-found/not-found-routing.module.ts

```diff
@@ -0,0 +1,16 @@
+import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+import { NotFoundComponent } from './not-found/not-found.component';
+
+const routes: Routes = [
+  {
+    path: '',
+    component: NotFoundComponent,
+  },
+];
+
+@NgModule({
+  imports: [RouterModule.forChild(routes)],
+  exports: [RouterModule],
+})
+export class NotFoundRoutingModule {}
```

Comment:
**explain why a new module is needed instead of a single component (not everything needs to be bundled into modules).
(IMO: remove module and add the page as a simple component)**